scarb/src/ops/new.rs

```diff
@@ -193,6 +199,19 @@ fn mk(
         "#})
     }
 
+    if snforge {
+        init_snforge(name)?;
+    }
+
+    Ok(())
+}
+
+fn init_snforge(name: PackageName) -> Result<()> {
+    Command::new("snforge")
```

Comment:
Shouldn't this inherit stdout/stderr? 
Shouldn't this get all env vars usually set by Scarb for subcommands (like `SCARB` so snforge does not call any other Scarb binary) - see https://github.com/software-mansion/scarb/blob/61dd36af1d28d60499832f06b3a5c9e83ba7647a/scarb/src/subcommands.rs#L11